### ISSUE TITLE
Lowfi Landscape: OngoingTopicListActivity

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#791]
Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/ce21cbf2-0439-47d7-bb37-005cfc6b9742/UP-Topics-in-Progress-

Also, please add test case for checking that item is visible in orientation change also. You will mainly need to do this in `OngoingTopicListActivityTest`

**Comment by rt4914** [comment]
Assigned to @virajpatel2411

**Comment by virajpatel2411** [comment]
@rt4914  Do I have to change anything in UI or I just have to add the test cases?

**Comment by rt4914** [comment]
> @rt4914 Do I have to change anything in UI or I just have to add the test cases?

You will need to create the landscape layout, currently app supports only portrait mode. Also, for landscape mode you will need add test cases also. Check implementation in other activities which have landscape mode.

**Comment by virajpatel2411** [comment]
Okaayy
And @rt4914  Where in the app will I find this page? I mean in the running app not in studio?

**Comment by rt4914** [comment]
Take latest, code from develop, inside Home - open navigationDrawer - Click on Profile - Click on `Topics in Progress`. 

Now, you will be able to see that screen only when there is some progress in the app which obviously does not exists right now. So you can do this copy [this code](https://gist.github.com/rt4914/5258375c8d2cca81b6a5b62ccab8e143) in `HomeFragmentPresenter` (Make sure this code is removed before you make a PR.)

Basically in this code we are using `storyProgressHelper` to mark fake progress in the app.

**Comment by virajpatel2411** [comment]
Okay
